### PR TITLE
[DD-972] added if/else function/component converting

### PR DESCRIPTION
**Pull request by boferri**

### CODE REVIEW

File: converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java [hunk 1]
```diff
@@ -880,4 +864,115 @@ private String getMAPIIdentifier(final MappingAttributePathInstance mapi) throws
...
+
+		final Iterator<String> iter = collectionSourceAttributes.iterator();
+
+		int i = 0;
+
+		while (iter.hasNext()) {
+
+			final String sourceAttribute = iter.next();
+
+			valueString += "${" + sourceAttribute + "}";
```
Comment (knutwalker) [inline converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java]: Maybe use a StringBuilder here?

File: converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java [hunk 2]
```diff
@@ -880,4 +864,115 @@ private String getMAPIIdentifier(final MappingAttributePathInstance mapi) throws
...
+
+			if ((i++ + 1) < collectionSourceAttributes.size()) {
+
+				valueString += delimiterString;
```
Comment (knutwalker) [inline converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java]: That is, for all this `valueString` appending

File: converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java (hunk 1 above)
Comment (boferri) [inline converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java]: yepp, can do so.

File: converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java (hunk 2 above)
Comment (boferri) [inline converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java]: yepp

**Comment by boferri** [comment]
see http://194.95.145.11/job/Github%20PR%20Datamanagement%20Platform/9/ for new test run

**Pull request merged**